AIDevGallery/Samples/WCRAPIs/BackgroundRemover.xaml

```diff
@@ -9,100 +9,72 @@
+                        <MenuFlyoutItem Click="PasteImage_Click" Text="From Clipboard" />
+                        <MenuFlyoutItem Click="LoadImage_Click" Text="From File" />
                     </MenuFlyout>
                 </DropDownButton.Flyout>
             </DropDownButton>
-            <StackPanel
-                x:Name="ActionsButtonPanel"
-                Orientation="Horizontal"
-                Spacing="12"
-                Visibility="Collapsed">
-                <Button
-                    Click="RemoveBackground_Click"
-                    Content="Remove background"
-                    Style="{StaticResource AccentButtonStyle}" />
-                <Button
-                    x:Name="CleanSelectionBtn"
-                    Click="CleanSelection_Click"
-                    Content="Clean selection"
-                    IsEnabled="False" />
-            </StackPanel>
         </StackPanel>
-
-        <TextBlock
-            x:Name="InstructionTxt"
-            Grid.Row="1"
-            Grid.ColumnSpan="2"
-            Margin="0,0,0,24"
-            Foreground="{ThemeResource TextFillColorSecondaryBrush}"
-            Style="{StaticResource CaptionTextBlockStyle}"
-            Text="Click on the parts of image that should be extracted, then click 'Remove background'. A maximum of 31 points can be selected."
-            TextWrapping="Wrap"
-            Visibility="Collapsed" />
-
-        <Grid
-            Grid.Row="2"
-            HorizontalAlignment="Left"
-            VerticalAlignment="Top">
-            <Image
-                x:Name="ImageSrc"
-                MaxHeight="360"
-                HorizontalAlignment="Stretch"
-                VerticalAlignment="Stretch"
-                Stretch="Uniform" />
-            <Canvas
-                x:Name="InputImageCanvas"
-                Width="{Binding Width, ElementName=ImageSrc}"
-                Height="{Binding Height, ElementName=ImageSrc}"
-                Background="Transparent"
-                PointerReleased="Canvas_PointerReleased"
-                SizeChanged="Canvas_SizeChanged" />
-        </Grid>
-        <Image
-            x:Name="ImageDst"
-            Grid.Row="2"
-            Grid.Column="1"
-            MaxHeight="360"
-            HorizontalAlignment="Left"
-            VerticalAlignment="Top"
-            Stretch="Uniform" />
-        <ProgressRing
-            x:Name="Loader"
+        <TextBlock 
+            x:Name="InstructionText"
             Grid.Row="2"
-            Grid.Column="1"
-            Width="48"
-            Height="48"
+            TextWrapping="WrapWholeWords"
+            FontSize="10"
             HorizontalAlignment="Center"
-            VerticalAlignment="Center"
-            IsActive="True"
-            Visibility="Collapsed" />
+            Text="Click on the image to select objects you wish to be included after background removal. 0 of 31 maximum points currently selected."/>
```

Comment:
Should we move this to the top?

There is now a lot going on at the bottom of the screen, with different fontsizes and texts
<img width="542" alt="image" src="https://github.com/user-attachments/assets/a3f98cd8-2b4f-4501-b6d6-6e6a473b85d8" />